### PR TITLE
Add Counterspell Dhaka

### PR DESCRIPTION
**Pull request by tamzid10001**

### CODE REVIEW

**Comment by tamzid10001** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Comments	Updated (UTC)
> **counterspell**	❌ Failed ([Inspect](https://vercel.com/hackclub/counterspell/13ZCncwBYzw26HL1H9hCJUuCh9Wf))			Oct 12, 2024 1:35pm

**Review by samdev-7 — APPROVED** [review]

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/dhaka**](https://counterspell-r2mullh2g.hackclub.dev/dhaka) |
|:-:|
|<a href="https://counterspell-r2mullh2g.hackclub.dev/dhaka">Screenshot of /dhaka</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/_ppfBfPHGBTIXQht8WJpUKdFWJBKubLSkvWpqPMC1EDhKXAfic4uucnGmYmWk4xVL3YVC5JWJzVJzyLhmu6uQQ.png">(view full size)</a> |







Commit 4513fa04dd56a3cdccf63ada7544d0d85e21ec70 (https://counterspell-r2mullh2g.hackclub.dev).

**Review by samdev-7 — CHANGES_REQUESTED** [review]
It seems like you copied the wrong folder, make sure you copy the ExampleCity folder and edit it from there.

File: components/Dhaka/About.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div className="space-y-12 max-w-8xl">
+        <h2 className="pb-6 text-4xl text-center uppercase retro text-whitepurp">
+          Make games, <span className="text-pink">meet cool people</span>
+        </h2>
+        <div className="grid grid-cols-1 xl:grid-cols-2 grid-rows-2 gap-12 leading-[3rem] tracking-wider mx-6 2xl:mx-0">
+          <div className="row-span-2 p-8 space-y-12 border-4 border-dashed border-pink md:p-12">
+            <p>
+              {/* TODO: Change `Example City` to the name of your city */}
+              Counterspell Dhaka is a beginner friendly game jam for high
+              schoolers happening in Example City where{" "}
```
Comment (samdev-7) [inline components/Dhaka/About.js:13]:
```suggestion
              schoolers happening in Dhaka where{" "}
```

File: components/Dhaka/Hero.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          <div className="hidden absolute top-[90%] lg:top-[145%] left-[9%] w-[80%] lg:scale-[130%] bg-[#090808] h-screen" />
+
+          <div className="flex items-center justify-center text-center">
+            <div className="z-50 absolute top-[43%] lg:top-[47%] text-[60%] sm:text-[80%] md:text-[100%] lg:text-[140%] tracking-wide">
+              <p className="text-[#CFD8ED] retro">
+                A BEGINNER FRIENDLY GAME JAM
+              </p>
+              <p className="text-pink retro text-[80%] sm:text-[90%] lg:text-[90%] mt-2">
+                {/* TODO: Replace `EXAMPLE CITY` with your city */}
+                NOVEMBER 23-24, 2024 • EXAMPLE CITY
```
Comment (samdev-7) [inline components/Dhaka/Hero.js:82]:
```suggestion
                NOVEMBER 23-24, 2024 • DHAKA
```

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost good to go. I've left some suggestions of edits you missed.

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**